lib/RexLint.php

```diff
@@ -40,18 +40,18 @@ public static function runFromWeb()
      */
     public static function getPathsToLint(): array
     {
-        $pathToLint = [
-            rex_path::src('addons/'),
-        ];
-        $modulesDir = DeveloperAddonIntegration::getModulesDir();
-        if ($modulesDir !== null) {
-            $pathToLint[] = $modulesDir;
-        }
+        $pathToLint = [];
+
+        foreach (RexStanUserConfig::getPaths() as $scanpath) {
+            $absPath = realpath(__DIR__.'/../'. $scanpath);
```

Comment:
@gharlan magst du mal testen ob die pfad auflösung auch für das yakamara dir layout funktioniert?

Reply:
Bekomme diese Exception:

**RuntimeException:** Could not resolve path: ../../../../src/modules/
**File:** src/addons/rexstan/lib/RexLint.php
**Line:** 49

<details>
<summary>Stacktrace</summary>

| Function                                     | File                                       | Line     |
| -------------------------------------------- | ------------------------------------------ | -------- |
| rexstan\RexLint::getPathsToLint              | src/addons/rexstan/lib/RexLint.php         | 65       |
| rexstan\RexLint::lintPaths                   | src/addons/rexstan/lib/RexLint.php         | 27       |
| rexstan\RexLint::runFromWeb                  | src/addons/rexstan/pages/lint-analysis.php | 9        |
| require                                      | src/core/lib/packages/package.php          | 280      |
| rex_package->includeFile                     | src/core/lib/be/controller.php             | 481      |
| rex_be_controller::{closure}                 | src/core/lib/util/timer.php                | 63       |
| rex_timer::measure                           | src/core/lib/be/controller.php             | 463      |
| rex_be_controller::includePath               | src/core/lib/be/controller.php             | 431      |
| rex_be_controller::includeCurrentPageSubPath | src/addons/rexstan/pages/index.php         | 9        |
| require                                      | src/core/lib/packages/package.php          | 280      |
| rex_package->includeFile                     | src/core/lib/be/controller.php             | 481      |
| rex_be_controller::{closure}                 | src/core/lib/util/timer.php                | 63       |
| rex_timer::measure                           | src/core/lib/be/controller.php             | 463      |
| rex_be_controller::includePath               | src/core/lib/be/controller.php             | 414      |
| rex_be_controller::includeCurrentPage        | src/core/backend.php                       | 239      |
| require                                      | src/core/boot.php                          | 143      |
| require                                      | public/redaxo/index.php                    | 13       |

</details>
<details>
<summary>System report (REDAXO 5.14.3, PHP 8.2.4, MariaDB 10.11.2)</summary>

| REDAXO        |            |
| ------------: | :--------- |
|       Version | 5.14.3     |


| PHP           |            |
| ------------: | :--------- |
|       Version | 8.2.4      |
|       OPcache | yes        |
|        Xdebug | yes        |


| Database      |                 |
| ------------: | :-------------- |
|       Version | MariaDB 10.11.2 |
| Character set | utf8mb4         |


| Server        |                |
| ------------: | :------------- |
|            OS | Darwin         |
|          SAPI | apache2handler |
|     Webserver | Apache/2.4.56  |


| Request       |             |
| ------------: | :---------- |
|       Browser | Safari/16.4 |
|      Protocol | HTTP/1.1    |
|         HTTPS | yes         |


| Packages            |            |
| ------------------: | :--------- |
|             adminer | 1.9.3      |
|              backup | 2.8.3      |
|            be_style | 3.1.2      |
| be_style/customizer | 3.1.2      |
|     be_style/redaxo | 3.1.2      |
|             cronjob | 2.9.0      |
|               debug | 1.2.2      |
|           developer | 3.9.2      |
|       documentation | dev        |
|             install | 2.10.1     |
|            markitup | 3.7.4      |
|              mblock | 3.4.12     |
|       media_manager | 2.13.3     |
|           mediapool | 2.12.1     |
|            metainfo | 2.9.1      |
|           phpmailer | 2.11.2     |
|             project | dev        |
|    quick_navigation | 6.0.2      |
|             rexstan | 1.0.109    |
|               sprog | 1.5.1      |
|           structure | 2.14.3     |
|   structure/content | 2.14.3     |
|                 url | 1.0.1      |
|               users | 2.9.2      |
|              watson | 2.3.0      |
|             yakicon | 1.0.0      |
|                ycom | 2.1-dev    |
|           ycom/auth | 2.1-dev    |
|          ycom/group | 2.1-dev    |
|             ydeploy | 1.2.0      |
|               yform | 2.3        |
|          yform/docs | 2.3        |
|         yform/email | 2.3        |
|       yform/manager | 2.3        |
|         yform/tools | 2.3        |
|         ynewsletter | 1.0        |
|            yrewrite | 2.5        |

</details>

Reply:
danke. könntest du nochmal probieren?